scripts/third_party_size_check.py

```diff
@@ -21,35 +21,54 @@
 
 import os
 import sys
+import yaml
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000
 
 
-def _check_size_in_dir(dir_path):
+def _get_skip_files_list():
+    """This lists the directories which should be skipped."""
+    with open('./app.yaml', 'r') as app_yaml:
+        try:
+            app_yaml_dict = yaml.safe_load(app_yaml)
+        except yaml.YAMLError as exc:
+            print exc
+        skip_files = app_yaml_dict.get('skip_files')
+
+        skip_files = [os.getcwd() + '/' + skip_files_dir
+                      for skip_files_dir in skip_files]
+
+    return skip_files
+
+
+def _check_size_in_dir(skip_files, dir_path):
     """Recursive method that checks the number of files inside the given
     directory.
-
     Args:
+         skip_files: str. The directory which files will not be counted.
          dir_path: str. The directory which files will be counted.
-
     Returns:
         The number of files inside the given directory.
     """
     number_of_files_in_dir = 0
     for name in os.listdir(dir_path):
+        if os.path.join(dir_path, name) in skip_files:
+            continue
         if os.path.isfile(os.path.join(dir_path, name)):
             number_of_files_in_dir += 1
         else:
             if os.path.isdir(os.path.join(dir_path, name)):
                 number_of_files_in_dir += _check_size_in_dir(
-                    os.path.join(dir_path, name))
+                    skip_files, os.path.join(dir_path, name))
     return number_of_files_in_dir
 
 
 def _check_third_party_size():
     """Checks if the third-party size limit has been exceeded."""
-    number_of_files_in_third_party = _check_size_in_dir(THIRD_PARTY_PATH)
+    skip_files = _get_skip_files_list()
```

Comment:
Okay, so rename the `skip_files` variable to `skip_files_list` here and everywhere in this file, where it is indeed a list.

Reply:
Done Thanks!